13LessonThirteen/in-class-project/index.html

```diff
@@ -1,10 +1,61 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://use.fontawesome.com/9d34820143.js"></script>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-9">
+                <img class="img-responsive" src="css/bg (2).jpg" />
```

Comment:
rename the image and reflect the changes here. Because as right now i can't see the image